### PR TITLE
Add new docs tip about passing multiple methods to `extend` and `override`

### PR DESCRIPTION
**Pull request by davwheat**
Depends on https://github.com/flarum/core/pull/2694

### CODE REVIEW

File: docs/extend/frontend.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ DiscussionPage.prototype.view = function() {
 }
 ```
 
-will turn Flarum's discussion pages into proclamations of "Hello World". How creative!
+...will turn Flarum's discussion pages into proclamations of "Hello World". How creative!
```
Comment (askvortsov1) [inline docs/extend/frontend.md:311]: Why this change?

**Review by askvortsov1 — APPROVED** [review]

File: docs/extend/frontend.md (hunk 1 above)
Comment (davwheat) [reply docs/extend/frontend.md:311]: Because I spent a few minutes trying to figure out why there was a lower case letter starting a sentence, until I noticed it was continuing the sentence above the code block.
